config/services/kube-supervisor.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<service>
+  <short>Kubernetes Supervisor Server</short>
```

Comment:
The docs you linked in the PR description call this a "Control-plane Node". Should we use that terminology? I did see the word "supervisor" anywhere in the linked docs.

Reply:
I'll switch to using `control-plane` and see how that looks.